src/plugin.ts

```diff
@@ -190,7 +190,7 @@ export class Spotify extends Plugin {
         if (typeof track.name !== "string") throw new TypeError(`The track name must be a string, received type ${typeof track.name}`);
 
         return {
-            title: track.name,
+            title: track.name + ` description:("Auto-generated by YouTube."),
```

Comment:
This breaks the entire file with the backtick, also I assume your adding a description property but this does nothing to help fix searching because Erela doesn't use a description to search, and the text inside of it I assume you are using as a search term and that also would do nothing to help fix this.

Reply:
oh, I forgot to add that backtick. but can you test it? because it is working fine for me.